packages/esm-commons-lib/src/components/encounter-list/encounter-list.component.tsx

```diff
@@ -67,7 +69,7 @@ export const EncounterList: React.FC<EncounterListProps> = ({
     onFormSave,
   } = useEncounterRows(patientUuid, encounterType, filter);
   const { moduleName, workspaceWindowSize, displayText, hideFormLauncher } = launchOptions;
-
+  
```

Comment:
Make sure your editor lints files on save and if configured correctly to avoid unnecessary changes like this extra space